docker/apache/openoni.conf

```diff
@@ -7,29 +10,12 @@ WSGIProcessGroup openoni-wsgi-app
 #ServerAdmin you@example.com
 #ServerName localhost
 
-WSGIScriptAlias / /opt/openoni/conf/openoni.wsgi
-
 RewriteEngine  on
 
 RewriteRule ^/share/(.*) http://loc.gov/share/$1 [P] 
 
 RewriteRule ^/data/(.+)/(.+)sample(.+) - [F]
 
-RewriteRule ^/index.html / [redirect=permanent,last]
```

Comment:
(maybe this is below, but note to self) if these are in the API path then we should change them!